Util/RealmAdmin.pm

```diff
@@ -237,6 +237,15 @@ sub leave_user {
     return;
 }
 
+sub reset_login {
```

Comment:
Noted, though including password was mostly for convenience for testing. I wouldn't expect (or condone) to be giving any passwords via email.

Reply:
In general, the tests can be modularized (combining reset_lock and reset_password into a single test function) to avoid this without adding options to the main code. We try to avoid (and don't always succeed) modifying the main code to have special cases for testing.